backend/open_webui/retrieval/loaders/main.py

```
@@ -222,6 +345,24 @@ def _get_loader(self, filename: str, file_content_type: str, file_path: str):
+            loader = MistralLoader(
+                api_key=self.kwargs.get("MISTRAL_OCR_API_KEY"), file_path=file_path
+            )
+        elif (
+            self.engine == "external"
+            and self.kwargs.get("MISTRAL_OCR_API_KEY") != ""
+            and file_ext
+            in ["pdf"]  # Mistral OCR currently only supports PDF and images
+        ):
+            loader = MistralLoader(
+                api_key=self.kwargs.get("MISTRAL_OCR_API_KEY"), file_path=file_path
+            )
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)